modules/View.js

```diff
@@ -145,6 +161,16 @@
     },
 
     /**
+     * Augments the context with custom content.
+     * @param context {Object} the context you can modify
+     * @return {Object} [Optional] If you return an object, it will be merged with the context
+     * @method _prepare
+     */
+    _prepare: function(context) {
```

Comment:
you could still use a _.noop here.

Reply:
I know, but I thought that adding this empty function would be easier for people looking to see how _prepare works (i.e. there's a context argument